src/app/ApplicationTemplate.Presentation/ViewModels/Diagnostics/Navigation/NavigationDebuggerViewModel.cs

```diff
@@ -1,13 +1,42 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Text;
+using Chinook.DynamicMvvm;
+using Chinook.SectionsNavigation;
+using Uno;
 
 namespace ApplicationTemplate.Presentation;
 
-public sealed class NavigationDebuggerViewModel : TabViewModel
+public sealed partial class NavigationDebuggerViewModel : TabViewModel
 {
+	[Inject] private ISectionsNavigator _sectionsNavigator;
+
 	public NavigationDebuggerViewModel()
 	{
 		Title = "Navigation";
 	}
+
+	/// <summary>
+	/// Gets a command that clears all modals and sections.
+	/// This should make the app look empty.
+	/// </summary>
+	public IDynamicCommand Clear => this.GetCommandFromTask(async ct =>
+	{
+		while (_sectionsNavigator.State.ActiveModal != null)
+		{
+			await _sectionsNavigator.CloseModal(ct);
+		}
+		foreach (var section in _sectionsNavigator.State.Sections)
+		{
+			await section.Value.Clear(ct);
+		}
+	});
+
+	/// <summary>
+	/// Gets a command that restores the app navigation to its initial state.
+	/// </summary>
+	public IDynamicCommand Restore => this.GetCommandFromTask(async ct =>
```

Comment:
Would Reinitialize be a better name than Restore since it's coming back to the initial state